### DISCUSSION TITLE
请问作者，你们能出一个表单设计器吗？ 还有就是工作流设计器，比如支持 activiti、flowable 这种的 使用bpmn.js

### DISCUSSION THREAD

**Discussion by lanjiachuan** [rsuite/rsuite#2675] [General]

**Comment by simonguo** [comment]
目前我们没有这方面的计划，但不排除我们会往这些领域提供一些指导，比如表单，你可以通过 [schema-form](https://github.com/rsuite/schema-form) 自动生成一个带有数据校验功能的表单，在此基础上只需要添加一个可以拖拽的库，通过拖拽后的结构生成 schema-form 所需要的数据，就实现了表单设计器的功能。

该 issue 将会被移动到 discussions
